apps/asn1pars.c

```diff
@@ -157,6 +157,8 @@ int asn1parse_main(int argc, char **argv)
             break;
         }
     }
+
+    /* No extra args. */
```

Comment:
I wonder if it should be printing 'reading from stdin' for the case when -in is not specified.

Reply:
I would prefer not to add verbosity like that; none of the other commands (e.g., x509 etc) do this.

Reply:
I was thinking from the perspective of a user who it presented with a cursor after doing this. Probably out of scope here..

Reply:
I'd recommend not adding this printing (if ever, it should be printed to stderr) as people definitely depend on using openssl as filter and not printing things that they do not expect print.